sdk-commons/src/main/java/org/openmrs/maven/plugins/model/BaseSdkProperties.java

```diff
@@ -1,333 +1,372 @@
+	}
+	
+	public void setName(String name) {
+		properties.setProperty("name", name);
+	}
+	
+	public List<Artifact> getModuleArtifacts() {
+		List<Artifact> artifactList = new ArrayList<>();
+		for (Object keyObject : getAllKeys()) {
+			String key = keyObject.toString();
+			String artifactType = getArtifactType(key);
+			if (artifactType.equals(TYPE_OMOD)) {
+				artifactList.add(new Artifact(checkIfOverwritten(key, ARTIFACT_ID), getParam(key),
+				        checkIfOverwritten(key, GROUP_ID), checkIfOverwritten(key, TYPE), "omod"));
+			}
+		}
+		return artifactList;
+	}
+	
+	public List<Artifact> getOwaArtifacts() {
+		List<Artifact> artifacts = new ArrayList<>();
+		for (Object keyObject : getAllKeys()) {
+			String key = keyObject.toString();
+			if (key.startsWith(TYPE_OWA + ".")) {
+				String artifactId = key.substring(TYPE_OWA.length() + 1);
+				artifacts.add(new Artifact(artifactId, getParam(key), Artifact.GROUP_OWA, Artifact.TYPE_ZIP));
+			}
+		}
+		return artifacts;
+	}
+	
+	public Map<String, String> getSpaProperties() {
+		Map<String, String> spaProperties = new HashMap<>();
+		for (Object keyObject : getAllKeys()) {
+			String key = keyObject.toString();
+			if (key.startsWith(TYPE_SPA + ".")) {
+				spaProperties.put(key.substring(TYPE_SPA.length() + 1), getParam(key));
+			}
+		}
+		return spaProperties;
+	}
+	
+	public List<Artifact> getWarArtifacts() {
+		List<Artifact> artifactList = new ArrayList<>();
+		for (Object keyObject : getAllKeys()) {
+			String key = keyObject.toString();
+			String artifactType = getArtifactType(key);
+			if (artifactType.equals(TYPE_WAR)) {
+				artifactList.add(new Artifact(checkIfOverwritten(key, ARTIFACT_ID), getParam(key),
+				        checkIfOverwritten(key, GROUP_ID), checkIfOverwritten(key, TYPE)));
+			}
+		}
+		return artifactList;
+	}
+	
 	public List<Artifact> getConfigArtifacts() {
 		List<Artifact> artifactList = new ArrayList<>();
 		for (Object keyObject : getAllKeys()) {
 			String key = keyObject.toString();
 			String artifactType = getArtifactType(key);
 			if (artifactType.equals(TYPE_CONFIG)) {
-				artifactList.add(
-						new Artifact(checkIfOverwritten(key, ARTIFACT_ID), getParam(key), checkIfOverwritten(key, GROUP_ID),
-								checkIfOverwritten(key, TYPE)));
+				artifactList.add(new Artifact(checkIfOverwritten(key, ARTIFACT_ID), getParam(key),
+				        checkIfOverwritten(key, GROUP_ID), checkIfOverwritten(key, TYPE)));
 			}
 		}
 		return artifactList;
 	}
-
+	
+	public List<Artifact> getContentArtifacts() {
+		List<Artifact> artifactList = new ArrayList<>();
+		for (Object keyObject : getAllKeys()) {
+			String key = keyObject.toString();
+			String artifactType = getArtifactType(key);
+			if (artifactType.equals(TYPE_CONTENT)) {
+				String artifactId = key.substring(TYPE_CONTENT.length() + 1);
+				System.out.println("n\n");
+				System.out.println(artifactId);
+				System.out.println(getParam(key));
+				System.out.println(Artifact.GROUP_CONTENT);
+				
```

Comment:
i think we can remove these `System.out.println`s during clean up after all the reviews